examples/lighting-app/nrfconnect/README.md

```diff
@@ -1,289 +1,338 @@
+            $ west update
 
-Download and install [GN meta-build system](https://gn.googlesource.com/gn/).
+3. Initialize environment variables referred to by the CHIP and the nRF Connect SDK build
+   scripts. Replace *nrfconnect-dir* with the path to your nRF Connect SDK
+   installation directory, and *toolchain-dir* with the path to GNU Arm
+   Embedded Toolchain.
 
-Make sure that you source the following file:
+        $ source nrfconnect-dir/zephyr/zephyr-env.sh
+        $ export ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
+        $ export GNUARMEMB_TOOLCHAIN_PATH=toolchain-dir
 
-        $ source <ncs-dir>/zephyr/zephyr-env.sh
+Now you can proceed with the [Building](#building) instruction.
 
-> **Note:**
->
-> Ensure that `$ZEPHYR_BASE`, `$GNUARMEMB_TOOLCHAIN_PATH`, and
-> `$ZEPHYR_TOOLCHAIN_VARIANT` environment variables are set in your current
-> terminal before building. `$GNUARMEMB_TOOLCHAIN_PATH` and
-> `$ZEPHYR_TOOLCHAIN_VARIANT` must be set manually.
+<hr>
 
-After your environment is set up, you are ready to build the example. The
-recommended tool for building and flashing the device is
-[west](https://docs.zephyrproject.org/latest/guides/west/).
+<a name="building"></a>
 
-The following commands will build the `lighting-app` example:
+## Building
 
-        $ cd ~/connectedhomeip/examples/lighting-app/nrfconnect
+Complete the following steps, regardless of the method used for setting up the environment:
 
-        # If this is a first time build or if `build` directory was deleted
-        $ west build -b nrf52840dk_nrf52840
+1. Navigate to the example's directory:
 
-        # Any subsequent build
-        $ west build
+        $ cd examples/lighting-app/nrfconnect
 
-After a successful build, the binary will be available under
-`<example-dir>/build/zephyr/zephyr.hex`
+2. Run the following command to build the example, with *board-name* replaced with the name
+   of the Nordic Semiconductor's board you own, for example `nrf52840dk_nrf52840`:
 
-### Supported nRF Connect SDK versions
+        $ west build -b board-name
 
-It is recommended to use the nRF Connect version which is being verified as a
-part of CHIP Continuous Integration testing, which happens to be `v1.4.0` at the
-moment. You may verify that the revision is used in
-[chip-build-nrf-platform](https://github.com/project-chip/connectedhomeip/blob/master/integrations/docker/images/chip-build-nrf-platform/Dockerfile)
-Docker image in case of doubt.
+   You only need to specify the board name on the first build.
+   See [Requirements](#requirements) for the names of compatible boards.
 
-Please refer to
-[this section](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#updating-the-repositories)
-in the user guide to learn how to update nRF Connect SDK repository. For example
-to checkout given `v1.4.0` revision the following commands should be called:
+3. Remove all build artifacts by running the following command:
```

Comment:
This cannot be point 3. Since only 1 and 2 are required. Point 3 shows only how to remove build directory before building application again with different configuration. So points 3 and 4 should/may be combined but not as a point 3 - maybe subsection (or whatever).

User should know that after point 2, he is ready to flash the device.